docs/Deploying-Push-Server.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: page
+title: Deploying Push Server
```

Comment:
We should prefer "github formatting" over this hacks for our documentation generator. We'll update that generator later. So, I recommend to use regular title for the page:
```
# Deploying Push Server
```

Reply:
Ok, I hope @joshis is fine with this change, there were some concerns previously :)